website/docs/integrations/overview.md

```diff
@@ -6,16 +6,16 @@ description: ConfigCat Integrations Overview. List of the available integrations
 Integrate ConfigCat with your technology stack and leverage all the benefits of Feature flags within your workflows.
 
 ### Slack
-Get updated via a Slack Channel message when someone changes a feature flag.
+Get notified in Slack when a feature flag changes.
 [Documentation](integrations/slack.md)
 ### Jira
-Turn features On / Off right from a linked Issue on your Jira board. Also you can add Targeting or Percentage Rules just as easily.
+Manage feature flags right from your Jira board.
```

Comment:
I see how the word "manage" is more precise here. However, for most users, turning features on and off, or doing a release is the most important use case. I think it would help users to manage "turn on/off" or "release".

Reply:
agreed